README.md

```diff
@@ -2,12 +2,47 @@
 
 ##### MENU
 
+* [Dependencies](#dependencies)
+* [How use hooks](#how-use-hooks)
 * [Style guide](#style-guide)
   * [Commit message format](#commit-message-format)
   * [Branch naming format](#branch-naming-format)
 * [Making changes to tracked branches](#making-changes-to-tracked-branches)
 * [Setup protectection for repo](#setup-protectection-for-repo)
 
+## Dependencies
+
+* sh
+* python 3.6+
+
+## How use hooks
+
+#### 1. Clone this repo
+
+```bash
+git clone git@github.com:pdffiller/styleguide-hooks.git
+```
+
+#### 2. Setup it as your template directory:
+
+```bash
+git config --global init.templatedir $(pwd)/styleguide-hooks
+```
+
+You can stop here **but**, it's not create symlink, so every time when "global" hooks is changed, you will need run `git init` in every repo. So, more usefull is
```

Comment:
I would replace it with the following text: "You may ignore the remainder nevertheless it's better to follow the next steps in order to avoid running 'git init' each time the 'global' hooks are changed or updated".